units/systemd-sysext-initrd.service

```diff
@@ -0,0 +1,34 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Commit message of the second commit:
`confexts to customize kernel modules` -> `sysexts to customize kernel modules` ?